ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1206,8 +1206,106 @@
                     fingerprint: false
                     only-if-success: false
       - '{name}-{test_name}-for-pull-request':
-          test_name: kind-aws-ipv6-only-networkpolicy
-          node: 'antrea-kind-test'
+          test_name: kind-cloud-e2e
```

Comment:
@KMAnju-2021 Since all jobs should be migrated to cloud jenkins, we can remove or migrate related items in projects-lab.yaml based on whether they have corresponding jobs in cloud YAML. For example, if we currently have both `kind-cloud-ipv6` and `kind-lab-ipv6`, we can remove `kind-lab-ipv6` to prevent duplicate jobs.